packages/@aws-cdk/custom-resource-handlers/lib/aws-eks/kubectl-handler/helm/__init__.py

```diff
@@ -90,7 +91,7 @@ def helm_handler(event, context):
             chart_dir = get_chart_from_oci(tmpdir.name, repository, version)
             chart = chart_dir
 
-        helm('upgrade', release, chart, repository, values_file, namespace, version, wait, timeout, create_namespace)
+        helm('upgrade', release, chart, repository, values_file, namespace, version, wait, timeout, create_namespace, atomic)
```

Comment:
Actually this will be passed to `helm` definition as `skip_crds` as that is the next parameter. I think
since `atomic` is a 12th parameter in the helm defintion, this needs to come after `skip_crds` is passed or you can pass the parameter explicitly like below

```helm('upgrade', release, chart, repository, values_file, namespace, version, wait, timeout, create_namespace, atomic=atomic)```

But not sure why the existing `skip_crds` is not passed to the defintion. Current code looks like it always be false and doesn't get from the resouce props. But this is different issue and just want to highlight it.

Reply:
yes it is interesting since both integration test and unit tests are passing for this too, imo it could be related to that these two flags don't need any additional handling in regards to values and will be just appended to command if set to true, somehow if this is propogating from props.atomic to underlying function, but yes this is weird that this is working for skip_crds and also have a successful tests passed

Reply:
Yeah, seems like the unit test for `skip_crds` to [disable by default](https://github.com/aws/aws-cdk/blob/eb82bc51b7b9c895a356575cbfeaa0aca42154cc/packages/aws-cdk-lib/aws-eks/test/helm-chart.test.ts#L262) is not correct I feel. Instead it needs to find the resource with `{ SkipCrds: true }` which will be expected to 0 like how you have handled for `atomic` or `wait` property.